src/System.Linq.Parallel/src/System/Linq/Parallel/Channels/SynchronousChannel.cs

```diff
@@ -29,15 +29,7 @@ internal sealed class SynchronousChannel<T>
 #if DEBUG
         // In debug builds, we keep track of when the producer is done (for asserts).
         private bool _done;
-#endif
-
-        //-----------------------------------------------------------------------------------
-        // Instantiates a new queue.
-        //
-
-        internal SynchronousChannel()
```

Comment:
Alright I'm not touching this then. I will recommit the other changes if they're ok?